src/utils/get-config.ts

```diff
@@ -80,27 +80,19 @@ export default async function getConfig(reconstruct?: true): Promise<Config> {
 		// Register Typescript compiler instance
 		let service: Service;
 
-		try {
-			service = require('ts-node').register({
-				compilerOptions: {
-					module: 'CommonJS',
-				},
-			});
-		} catch (e) {
-			if (e.code === 'MODULE_NOT_FOUND') {
-				throw new Error('To use a Typescript config file, ts-node should be installed as a dev-dependency');
-			}
-
-			throw e;
-		}
+		service = require('ts-node').register({
```

Comment:
Is this how you're supposed to import `ts-node`? Instead of something like `import tsNode from 'ts-node';` at the top?